backend/coverage/coverage-summary.json

```diff
@@ -0,0 +1,13 @@
+{"total": {"lines":{"total":1403,"covered":754,"skipped":0,"pct":53.74},"statements":{"total":1403,"covered":754,"skipped":0,"pct":53.74},"functions":{"total":56,"covered":23,"skipped":0,"pct":41.07},"branches":{"total":65,"covered":46,"skipped":0,"pct":70.77}}
```

Comment:
Can we add this file to the .gitignore, or is this necessary for the coverage badge?
When I think of code coverage I see two primary use cases:

1. As a developer, I want to view the coverage of a new test I added.
2. As a developer, I want to be notified if changes reduce the code coverage beyond a given threshold. For example, a build should fail if the coverage is less than x%.